OpenSSL-Strategy.md

```diff
@@ -123,7 +123,12 @@ Node.js EOL dates:
 
 ## Node.js version 15.x (est. Oct 2020) (EOL Jun 2021)
 
-* OpenSSL version: 1.1.1
+* [quictls/OpenSSL](https://github.com/quictls/openssl/tree/OpenSSL_1_1_1j+quic)
+version: 1.1.1j+quic
```

Comment:
This appears to be [outdated](https://github.com/nodejs/node/pull/38084) already, maybe we don't need to mention the specific version.

Reply:
Yep, perhaps instead make mention that the version is identified by branch name?

Reply:
Yeah, that sounds good. I'll update this tomorrow. Thanks!